src/golang/lib/logging/response.go

```diff
@@ -51,22 +52,37 @@ func LogRoute(
 	status := "SUCCEEDED"
 	var errMsg string
 	if err != nil {
-		status = "ERROR"
+		status = errorStatus
 		errMsg = err.Error()
 	}
 
-	log.WithFields(log.Fields{
-		"ServiceName":   serviceName,
-		"URL":           r.URL,
-		"Headers":       headers,
-		"Status":        status,
-		"Code":          statusCode,
-		"Component":     component,
-		"Route":         routeName,
-		"UserId":        ctx.Value(aq_context.UserIdKey),
-		"UserRequestId": ctx.Value(aq_context.UserRequestIdKey),
-		"Error":         errMsg,
-	}).Info()
+	if status == errorStatus {
+		log.WithFields(log.Fields{
```

Comment:
Style nit: Is this code deduplictable? i.e., we can do `logMsg := log.WithFields(...)` and then do `.Error()` or `.Info()` based on the status?